src/d/d_menu_collect.cpp

```diff
@@ -5,15 +5,323 @@
+        'ci04', 'ci03', 'ci02', 'ci01', 'ci00',
+    };
+
+    static const u32 l_htp[] = {
+        'htp2', 'htp1', 'htp3',
+    };
+
+    static const u32 l_tit[] = {
+        'it08', 'it09', 'it10', 'it11', 'it12', 'it13',
+    };
+
+    static const u32 l_tik[] = {
+        'ik08', 'ik09', 'ik10', 'ik11', 'ik12', 'ik13'
+    };
+
+    static const u32 l_cn[] = {
+        'cn01', 'cn02', 'cn03', 'cn04', 'cn05', 'cn06'
+    };
+
+    static const u32 l_wn[] = {
+        'wn01', 'wn02', 'wn03', 'wn04', 'wn05', 'wn06'
+    };
+
+    static const u32 l_i0[] = {
+        'i011', 'i021', 'i031', 'i041', 'i051',
+        'i061'
+    };
+
+    static const u32 l_i1[] = {
+        'i012', 'i022', 'i032', 'i042', 'i052', 'i062'
+    };
+
+    static const u32 l_bs[] = {
+        'bs01', 'bs02', 'bs03', 'bs04', 'bs05', 'bs06'
+    };
+
+    static const u32 l_hmb[] = {
+        'hmb1', 'hmb2', 'hmb3', 'hmb4', 'hmb5', 'hmb6'
+    };
+
+    static const u32 l_wit[] = {
+        'it00', 'it01', 'it02', 'it03', 'it04'
+    };
+
+    static const u32 l_wik[] = {
+        'ik00', 'ik01', 'ik02', 'ik03', 'ik04'
+    };
+
+    static const u32 l_ip[] = {
+        'ip09', 'ip07', 'ip05', 'ip03', 'ip01'
+    };
+
+    static const u32 l_wk[] = {
+        'wk00', 'wk01', 'wk02', 'wk03',
+    };
+
+    static char* triTex[8] = {
+        "triforce_00.bti", "triforce_01.bti", "triforce_02.bti", "triforce_03.bti", 
+        "triforce_04.bti", "triforce_05.bti", "triforce_06.bti", "triforce_07.bti",
+    };
+    static char* symTex[3] = {
+        "god_symbol_02.bti", "god_symbol_00.bti", "god_symbol_01.bti",
+    };
+    static char* wepTex[5] = {
+        "sword_00.bti", "shield_00.bti", "gloves_00.bti", "amulet_00.bti", "amulet_01.bti",
+    };
+
+    J2DPane* mPane = m004->search('ROOT');
+    m2478 = mPane;
+
+    fopMsgM_setPaneData(&m008, m004, 'ft15');
+    fopMsgM_setPaneData(&m040, m004, 'ft16');
+    fopMsgM_setPaneData(&m078, m004, 'ft17');
+
+    for (int i = 0; i < (s32)ARRAY_SIZE(m0B0); i++) {
+        fopMsgM_setPaneData(&m0B0[i], m004, l_ft[i]);
+        fopMsgM_setPaneData(&m3F8[i], m004, l_fd[i]);
+
+        m0B0[i].mUserArea = 0xFFFF;
+    }
+
+    fopMsgM_setPaneData(&m740, m004, 'str0');
+    fopMsgM_setPaneData(&m778, m004, 'st00');
+    fopMsgM_setPaneData(&m7B0, m004, 'nt00');
+    fopMsgM_setPaneData(&m7E8, m004, 'nk00');
+    fopMsgM_setPaneData(&m820, m004, 'no11');
+
+    m820.mUserArea = m820.pane->mRotation;
+
+    if (m820.mUserArea > 180) m820.mUserArea -= 360;
+
+    ((J2DTextBox*)(m740).pane)->setFont(m2474);
+    ((J2DTextBox*)(m778).pane)->setFont(m2470);
+
+    // Couple of functions missing here
+    ((J2DTextBox*)(m740).pane)->setWhite(-1);
+    ((J2DTextBox*)(m740).pane)->setCharColor(-1);
+    ((J2DTextBox*)(m740).pane)->setGradColor(-1);
+
+    ((J2DTextBox*)(m778).pane)->setWhite(-1);
+    ((J2DTextBox*)(m778).pane)->setCharColor(-1);
+    ((J2DTextBox*)(m778).pane)->setGradColor(-1);
```

Comment:
I meant the opposite, so it shows as a hex color, like this:
```suggestion
    ((J2DTextBox*)(m740).pane)->setWhite(0xFFFFFFFF);
    ((J2DTextBox*)(m740).pane)->setCharColor(0xFFFFFFFF);
    ((J2DTextBox*)(m740).pane)->setGradColor(0xFFFFFFFF);

    ((J2DTextBox*)(m778).pane)->setWhite(0xFFFFFFFF);
    ((J2DTextBox*)(m778).pane)->setCharColor(0xFFFFFFFF);
    ((J2DTextBox*)(m778).pane)->setGradColor(0xFFFFFFFF);
```
Or does it not match written as a positive number?